build.gradle

```diff
@@ -35,16 +35,16 @@ task clean(type: Delete) {
 ext {
```

Comment:
I'm guessing one of these introduce any breaking changes?

Reply:
Not seeing any breaking changes from testing or CI. Some of the libraries are on older versions because the new ones have breaking changes, if that was what you meant.